front/app/api/comments_by_project/useCommentsByProject.ts

```diff
@@ -0,0 +1,34 @@
+import { useQuery } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+import fetcher from 'utils/cl-react-query/fetcher';
+import {
+  ICommentsByProject,
+  CommentsByProjectKeys,
+  ICommentsByProjectParams,
+} from './types';
+import commentsByProjectKeys from './keys';
+
+const fetchCommentsByProject = (params: ICommentsByProjectParams) =>
+  fetcher<ICommentsByProject>({
+    path: `/stats/comments_by_project`,
```

Comment:
Minor for consistency. If we change this, we can do the same for other instances in this PR
```suggestion
    path: '/stats/comments_by_project',
```

Reply:
I don't really see an issue with leaving these as they are :))